src/server/service/table_service.cpp

```diff
@@ -61,6 +61,11 @@ std::string getWrongOptionHelp(const engine::api::TableParameters &parameters)
         help = "fallback_speed must be > 0";
     }
 
+    if (parameters.scale_factor < 0)
```

Comment:
This tests for < 0 but the error message says it must be > 0.

The `if` should probably be `<=` to match the message.